docs/spikes/2020-04-how-to-migrate-lb3-components.md

```diff
@@ -130,26 +130,25 @@ coming from._
 17. Add a new local service (a class providing JS/TS API), e.g. Push service or
     (database) Migration service.
 
-    - The service must be configurable by the target app (e.g. Apple & Google
-      credentials for push notifications, transport config for logger
-      frameworks, Amazon S3 credentials for storage component, etc.).
+    The service must be configurable by the target app (e.g. Apple & Google
+    credentials for push notifications, transport config for logger frameworks,
+    Amazon S3 credentials for storage component, etc.).
 
-      _LB3 typically uses component configuration. I think in LB4, we can use
-      `@configure` specific to each service (see the logger extension for an
-      example)._
+    _LB3 typically uses component configuration. I think in LB4, we can use
+    `@configure` specific to each service (see the logger extension for an
+    example)._
 
 18. Inject LB3 models (LB4 entities & repositories) to the service. These models
     can be provided either by the extension or by the target application. When
     using models from the target application, the developer needs an option to
     specify which models (entities) and associated repositories to use.
 
-    _Note: In LB4, models/entities are not injectable now, we used them
-    primarily for the type information (to describe the shape of model data). As
-    part of the follow-up research, we will need to decide if it's enough to
-    inject LB4 Repository to a component or if we need to make model/entity
-    classes injectable too. Also note that `DefaultCrudReposiory` class provides
-    a public instance property `entityClass` referencing the model/entity it's
-    bound to._
+    _Note: In LB4, models/entities are used primarily for the type information
+    (to describe the shape of model data). As part of the follow-up research, we
+    will need to decide if it's enough to inject LB4 Repository to a component
+    or if we need to inject model/entity classes too. Also note that
+    `DefaultCrudReposiory` class provides a public instance property
+    `entityClass` referencing the model/entity it's bound to._
```

Comment:
Updated the text to accommodate https://github.com/strongloop/loopback-next/pull/5361